File: server/csharp/MetaObjects.Render/Recover/Tolerance.cs
namespace MetaObjects.Render.Recover;

/// <summary>
/// STRICT: case-sensitive, minimal repair.
/// NORMAL: case-insensitive keys/tags (default).
/// LOOSE: reserved for future maximal-repair behavior (currently behaves identically to NORMAL).
/// </summary>
public enum Tolerance
{
    Strict,
    Normal,
    Loose,
}

File: server/java/render/src/main/java/com/metaobjects/render/extract/Tolerance.java
package com.metaobjects.render.extract;

/** STRICT: case-sensitive, minimal repair. NORMAL: case-insensitive keys/tags (default). LOOSE: maximal repair. */
public enum Tolerance {
    STRICT,
    NORMAL,
    // NOTE: LOOSE currently behaves identically to NORMAL (case-insensitive). Reserved for future maximal-repair behavior.
    LOOSE
}
